app/javascript/controllers/update_status_controller.js

```diff
@@ -17,21 +23,41 @@ export default class extends Controller {
     currentStatus: String
   }
 
-  // If a member clicks the "Update Status" button, open the modal
+  // If a captain clicks the "Update Status" button, open the modal
   // If they click the "Cancel" button, the "X" button, or anywhere on the
   // modal overlay, close the modal then de-select all inputs and clear the
   // flash message from it.
-  toggleModal() {
-    if (this.modalTarget.classList.contains("hidden")) {
-      this.modalTarget.classList.remove("hidden");
+  toggleCaptainModal() {
+    if (this.captainModalTarget.classList.contains("hidden")) {
+      this.captainModalTarget.classList.remove("hidden");
       this.selectCurrentStatus();
     } else {
-      this.modalTarget.classList.add("hidden");
+      this.captainModalTarget.classList.add("hidden");
       this.deselectAllInputs();
       this.clearFlashMessage();
     }
   }
 
+  // When a member clicks the "Mark Complete" button, open a modal
+  // that asks them if the assignment is totally complete. Make the user
```

Comment:
I like this, but it will probably be obnoxious if they get asked this every time. Maybe it gets asked once per chore? That may not be the right thing, but it shouldn't be every time I think.